staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_status.go

```diff
@@ -168,6 +168,13 @@ func (o *RolloutStatusOptions) Run() error {
 		Flatten().
 		Do()
 
+	info, _ := r.Infos()
+
+	if len(info) == 0 {
+		fmt.Fprintf(o.Out, "no resources found in %s namespace.\n", o.Namespace)
```

Comment:
I think this should be to o.ErrOut similar to https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/get/get.go#L565